CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Draft
+### Fixed
+- Fixed password complexity error not displaying the complexity rules set in the store settings
```

Comment:
Can you please update this to be consisent with other items in the file?

Reply:
@yurytut1993 Fixed, let me know if it looks good!

Reply:
I mean you should add link to your pr like
`- Fixed password complexity error not displaying the complexity rules set in the store settings [#2117](https://github.com/bigcommerce/cornerstone/pull/2117)`

Reply:
@yurytut1993 Ahh okay, sorry about that! Fixed. These PRs are a little different from the other ones I've done :)

Reply:
No problem:) resolved